src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ReplyToMessage.vue

```diff
@@ -93,33 +96,31 @@ export default defineComponent({
   },
   data(): ReplyToMessageData {
     return {
-      text: " ",
+      text: "",
       Rules,
       showCloseDialog: false,
       areAttachedFilesValid: true,
       isFileUploadInProgress: false,
+      formValid: false,
     };
   },
   methods: {
     async handleReplyToMessage() {
-      if (!this.isFileUploadInProgress) {
-        if (this.areAttachedFilesValid) {
-          if (this.text.trim().length > 0) {
-            const { error } = await sendMessage({ communication: { id: this.messageId, text: this.text } });
-            if (error) {
-              this.alertStore.setFailureAlert("Sorry, something went wrong and your changes could not be saved. Try again later.");
-            } else {
-              this.alertStore.setSuccessAlert("Message sent successfully.");
-              this.router.push("/messages");
-            }
-          } else {
-            this.alertStore.setFailureAlert("You must enter all required fields in the valid format to continue.");
-          }
+      const { replyFormValid } = await (this.$refs.replyForm as any).validate();
```

Comment:
we can use VForm for the type. 
Ex. https://github.com/bcgov/ECC-ECER/blob/0b09bad73b9843a7d6117c32b00bbe5064283f6c/src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/CertificationType.vue#L122

does { replyFormValid } = await blah.validate(); work? 

I thought validate just returns a "valid" or "errors" property. So I think replyFormValid would always be undefined. 
https://vuetifyjs.com/en/api/v-form/#usage:~:text=validate%3A%20()%20%3D%3E%20Promise%3C%7B

Reply:
Nice catch! fixed!